lambdas/drivingpermitcheck/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/service/dvla/endpoints/TokenHttpRetryStatusConfig.java

```diff
@@ -0,0 +1,75 @@
+package uk.gov.di.ipv.cri.drivingpermit.api.service.dvla.endpoints;
+
+import uk.gov.di.ipv.cri.drivingpermit.api.service.HttpRetryStatusConfig;
+
+import java.util.List;
+
+import static uk.gov.di.ipv.cri.drivingpermit.api.service.dvla.endpoints.ResponseStatusCodes.BAD_REQUEST;
+import static uk.gov.di.ipv.cri.drivingpermit.api.service.dvla.endpoints.ResponseStatusCodes.FORBIDDEN;
+import static uk.gov.di.ipv.cri.drivingpermit.api.service.dvla.endpoints.ResponseStatusCodes.SUCCESS;
+import static uk.gov.di.ipv.cri.drivingpermit.api.service.dvla.endpoints.ResponseStatusCodes.UNAUTHORISED;
+import static uk.gov.di.ipv.cri.drivingpermit.library.metrics.ThirdPartyAPIEndpointMetric.DVLA_TOKEN_HTTP_RETRYER_REQUEST_SEND_FAIL;
+import static uk.gov.di.ipv.cri.drivingpermit.library.metrics.ThirdPartyAPIEndpointMetric.DVLA_TOKEN_HTTP_RETRYER_REQUEST_SEND_OK;
+import static uk.gov.di.ipv.cri.drivingpermit.library.metrics.ThirdPartyAPIEndpointMetric.DVLA_TOKEN_HTTP_RETRYER_REQUEST_SEND_RETRY;
+import static uk.gov.di.ipv.cri.drivingpermit.library.metrics.ThirdPartyAPIEndpointMetric.DVLA_TOKEN_HTTP_RETRYER_SEND_ERROR;
+import static uk.gov.di.ipv.cri.drivingpermit.library.metrics.ThirdPartyAPIEndpointMetric.DVLA_TOKEN_HTTP_RETRYER_SEND_MAX_RETRIES;
+
+public class TokenHttpRetryStatusConfig implements HttpRetryStatusConfig {
+
+    private final List<Integer> neverRetryCodes =
+            List.of(SUCCESS, BAD_REQUEST, UNAUTHORISED, FORBIDDEN);
+
+    private static final int TOO_MANY_REQUESTS = 429;
+    private final List<Integer> retryCodes = List.of(TOO_MANY_REQUESTS);
```

Comment:
Should this have the same retry codes as driver match?

Reply:
404 is retryable for match due to the api, but not for the token.